test/NavbarSpec.js

```diff
@@ -61,7 +61,7 @@ describe('Nav', function () {
     let instance = ReactTestUtils.renderIntoDocument(
       <Navbar componentClass={'header'}/>
     );
-    assert.ok(React.findDOMNode(instance).nodeName, 'HEADER');
+    assert.equal(React.findDOMNode(instance).nodeName, 'HEADER');
```

Comment:
Yes, I've came across this issue as I looked how other components are testing the `componentClass` property.

I’ve made it a separate commit. If desired I can make a separate PR for this change.